recipes/_software.rb

```diff
@@ -27,11 +27,17 @@
 
 # Debian Stretch needs some packages which are no longer installed by default
 stretch_packages = %w{
-  dirmngr
   gnupg
 }
 
 package stretch_packages do
   action :upgrade
   only_if { node[:platform] == "debian" and node[:platform_version].to_i >= 9 }
 end
+
+# Remove dirmngr even though it seems to be used by apt-key.
```

Comment:
This explicit uninstall would run on every chef-client execution. So if we choose to use dirmngr on those hosts needing it for apt-key we have to remove this code again. I'd rather vote to just not install but not explicitly uninstall dirmngr in this recipe. As it was only one host showing the many processes we should rather uninstall it there manually in my opinion.

Reply:
That's ok with me. Will take care...